src/modules/chatExtras/i18n/it_IT.rot.json

```diff
@@ -0,0 +1,10 @@
+{
+	"description": "Mostra la data e l\'ora anziché solo l\'ora",
```

Comment:
escapes are not needed for a **'** when using **"**

Reply:
yes when i have rename the file (wrong typo)i has changed ;)

Reply:
oh wait, you already changed that